src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/ImplementIDisposableCorrectly.cs

```diff
@@ -532,7 +533,7 @@ private bool ValidateOperations(ImmutableArray<IOperation> operations)
             {
                 foreach (IOperation operation in operations)
                 {
-                    if (!operation.IsImplicit && !ValidateOperation(operation))
+                    if (!operation.IsImplicit && operation.Kind != OperationKindEx.Attribute && !ValidateOperation(operation))
```

Comment:
Do we need new tests for this change?

Reply:
There were existing test baselines preserved for all the changes except where I added tests.